### PR TITLE
Removed clear-session

### PR DESCRIPTION
**Pull request by ianmagenta**
## Summary
Remove the `clear-session` from the Cerner deep linking URL.
Note: I opted to leave in the `useSingleLogoutPaths` path, despite being redundant. This function is called in several places and I figured it would be best to maintain the function's interface (in case it changes again in the future).

## Related issue(s)
https://github.com/department-of-veterans-affairs/va.gov-team/issues/70106

## What areas of the site does it impact?
This primarily impacts the following components which all call `getCernerURL`:
- `CernerWidgets`
- `CernerAlert`
- `ScheduleCernerPage`
- `FacilitiesRadioWidget`

## Acceptance criteria
- [x] Remove the `clear-session` from the Cerner utilities file in `vets-website`
- [x] Update unit tests if required

### CODE REVIEW

**Review by powellkerry — APPROVED** [review]

**Comment by joeniquette** [comment]
@ianmagenta can you please check the deeplinks to see what they list for each one, both va.gov and mhv, staging and prod. Once documented then lets make a call as to if its really safe to remove clear session.

File: src/applications/vaos/tests/new-appointment/components/VAFacilityPage/index.unit.spec.jsx [hunk 1]
```diff
@@ -1373,7 +1373,7 @@ describe('VAOS <VAFacilityPage>', () => {
         within(cernerSiteLabel)
           .getByRole('link', { name: /My VA Health/ })
           .getAttribute('href'),
-      ).to.contain('pages%2Fscheduling%2Fupcoming');
+      ).to.contain('pages/scheduling/upcoming');
```
Comment (asg5704) [inline src/applications/vaos/tests/new-appointment/components/VAFacilityPage/index.unit.spec.jsx:1376]: Revert this

File: src/applications/vaos/tests/new-appointment/components/ScheduleCernerPage.unit.spec.jsx [hunk 2]
```diff
@@ -169,7 +169,7 @@ describe('VAOS <ScheduleCernerPage>', () => {
         screen
           .getByRole('link', { name: /My VA Health/i })
           .getAttribute('href'),
-      ).to.contain('pages%2Fscheduling%2Fupcoming');
+      ).to.contain('pages/scheduling/upcoming');
```
Comment (asg5704) [inline src/applications/vaos/tests/new-appointment/components/ScheduleCernerPage.unit.spec.jsx:172]: Revert this

File: src/platform/utilities/cerner/index.js [hunk 3]
```diff
@@ -10,9 +10,7 @@ export const getCernerURL = (path, useSingleLogoutPaths = false) => {
     return `${host}${path}?authenticated=true`;
   }
 
-  return `${host}/clear-session?to=${encodeURIComponent(
-    `${host}${path}?authenticated=true`,
-  )}`;
+  return `${host}${path}?authenticated=true`;
```
Comment (asg5704) [inline src/platform/utilities/cerner/index.js:13]: Can you wrap this in an `encodeURIComponent`

**Review by asg5704 — CHANGES_REQUESTED** [review]
Can you make the requested changes and I can approve. Because running the unit test locally => 110 failed tests.

**Comment by ianmagenta** [comment]
@asg5704 - Good to go!

**Review by asg5704 — APPROVED** [review]
Thank you

**Pull request merged**